### PR TITLE
identity, refactoring, removed TestUntil (sync with 0.7.9)

### PR DESCRIPTION
**Pull request by zpbappi**

### CODE REVIEW

**Comment by VaughnVernon** [comment]
@zpbappi Not sure why AppVeyor is failing. The PRs seem to fail often on AppVeyor. Is this unusual?

**Comment by zpbappi** [comment]
@VaughnVernon the test fails randomly. we believe it is due to concurrency not handled properly somewhere. We are tracking this as an issue [here](https://github.com/vlingo-net/vlingo-net-common/issues/18). @tjaskula or I will look at it soon.

File: src/Vlingo.Common/BasicCompletes.cs [hunk 1]
```diff
@@ -97,7 +97,7 @@ public virtual ICompletes<T> AndThenConsume(T failedOutcomeValue, System.Action<
 
         public virtual ICompletes<T> AndThenConsume(System.Action<T> consumer) => AndThenConsume(-1L, default(T), consumer);
 
-        public virtual O AndThenInto<F, O>(long timeout, F failedOutcomeValue, Func<T, O> function)
```
Comment (tjaskula) [inline src/Vlingo.Common/BasicCompletes.cs]: In .NET Api it is more idiomatic to use TimeSpan or int32 but I guess it was to comply to Java version

File: src/Vlingo.Common/Identity/IIdentityGenerator.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+using System;
+using System.Security.Cryptography;
+using Vlingo.UUID;
+
+namespace Vlingo.Common.Identity
+{
+    public interface IIdentityGenerator
```
Comment (tjaskula) [inline src/Vlingo.Common/Identity/IIdentityGenerator.cs]: My personal preference is to keep 1 class/interface per file but I'm not suggesting it's not ok like that

Comment (zpbappi) [reply src/Vlingo.Common/Identity/IIdentityGenerator.cs]: I prefer that too. Sometimes, I feel just too lazy for internal classes. 😛

**Pull request merged**